### DISCUSSION TITLE
AdGuard Home in OpenWrt: PTR queries

### DISCUSSION THREAD

**Discussion by ali27868** [AdguardTeam/AdGuardHome#3857] [Q&A] [answered]
Hi!

Can you please suggest what is the proper way to set up in AdGuard Home the DNS setting for PTR queries (servers, necessary checkmarks). Unfortunately, not so obvious.

AGH is set up on OpenWrt router, where router does DHCP, not AGH and DNS requests are forwarded to AGH.

Thank you.

**Comment by ainar-g** [comment]
Hello.  What do you mean by “set up for PTR queries”?  Can you provide an example of a query, for example using `nslookup` or `dig`, that doesn't work for you currently?

**Reply by ali27868** [reply]
Hi!
I am looking for general information how to set up "Private reverse DNS servers" in AGH - guide, if you wish.
If there is none, then specifically - How do I make sure that local LAN requests do not go from AGH to upstream servers? Perhaps, a preceding question is How to make sure local LAN requests do not go to AGH?
Thank you.

**Reply by ainar-g** [reply, marked as answer]
Without an example of a query it's a bit hard to get, but if you want requests like:

```sh
nslookup 192.168.12.34
```

to be processed by a local DNS server 192.168.10.1 then put the following line to the _Private reverse DNS servers_ input:

```none
192.168.10.1
```

**Reply by ali27868** [reply]
Thanks, this answers my question.
